atomics/T1204.002/T1204.002.yaml

```diff
@@ -321,3 +321,71 @@ atomic_tests:
     cleanup_command: |
       Remove-Item #{pua_file}
 
+- name: Office Generic Payload Download
+  description: |
+    This Test uses a VBA macro to launch Powershell which will download a file from a user defined web server.
+    Required input agruments are c2_domain, c2_parent_directory, file_name
```

Comment:
Can you use defaults for these parameters that will work for most people "out of the box" such as downloading a file from the atomic red team repo that is known to exist?